### PR TITLE
Updates for 0.2 release

### PR DESCRIPTION
**Pull request by sameerz**
Signed-off-by: Sameer Raheja <sraheja@nvidia.com>

@mengdong @GaryShen2008 can you review these updates for the google init script and see whether anything is missing?

I think the README.md section on Spark 2.3 with XGBoost needs to be reviewed carefully.  

I am making these changes in my own fork, if these look good, @mengdong we need to create a PR to the google initialization actions repository.

### CODE REVIEW

File: rapids/rapids.sh [hunk 1]
```diff
@@ -30,7 +30,7 @@ readonly RUN_WORKER_ON_MASTER=$(get_metadata_attribute 'dask-cuda-worker-on-mast
 readonly DEFAULT_CUDA_VERSION="10.2"
```
Comment (mengdong) [inline rapids/rapids.sh]: I wonder if we should select cuda 11 as default. I am making the changes in GPU driver as well, there is an on-going pull request and I need cover a test failure. But ampere should reach dataproc soon.

Comment (sameerz) [reply rapids/rapids.sh]: Once you test cuda 11 on Dataproc and confirm it works we should update.

Comment (mengdong) [reply rapids/rapids.sh]: I have run a simple GPU job with cuda 11 on dataproc and it execute on GPU
```
== Physical Plan ==                                                   
*(3) GpuColumnarToRow false                                           
+- GpuShuffledHashJoin [_1#0L], [_1#2L], Inner, BuildRight, false     
   :- GpuCoalesceBatches TargetSize(2147483647)                       
   :  +- GpuColumnarExchange gpuhashpartitioning(_1#0L, 48), [id=#230]
   :     +- GpuCoalesceBatches TargetSize(2147483647)                 
   :        +- GpuFilter gpuisnotnull(_1#0L)                          
   :           +- GpuRowToColumnar TargetSize(2147483647)             
   :              +- *(1) Scan ExistingRDD[_1#0L]                     
   +- GpuCoalesceBatches RequireSingleBatch                           
      +- GpuColumnarExchange gpuhashpartitioning(_1#2L, 48), [id=#236]
         +- GpuCoalesceBatches TargetSize(2147483647)                 
            +- GpuFilter gpuisnotnull(_1#2L)                          
               +- GpuRowToColumnar TargetSize(2147483647)             
                  +- *(2) Scan ExistingRDD[_1#2L]
```

File: rapids/rapids.sh [hunk 2]
```diff
@@ -69,6 +69,12 @@ function install_spark_rapids() {
     local -r cudf_cuda_version="${CUDA_VERSION//\./-}"
   fi
 
+  if [[ "${CUDA_VERSION}" == "11.0" ]]; then
+    local -r cudf_cuda_version="11"
+  else
+    local -r cudf_cuda_version="${CUDA_VERSION//\./-}"
+  fi
+
```
Comment (mengdong) [inline rapids/rapids.sh:68]: maybe do
```
  if [[ "${CUDA_VERSION}" == "10.0" ]]; then
    local -r cudf_cuda_version="10"
  elif [[ "${CUDA_VERSION}" == "11.0" ]]; then
    local -r cudf_cuda_version="11"
  else
    local -r cudf_cuda_version="${CUDA_VERSION//\./-}"
  fi
```

**Comment by mengdong** [comment]
Also I wonder if we should add following to spark-defaults if release version is 0.2
```
spark.sql.extensions=com.nvidia.spark.udf.Plugin
spark.rapids.sql.udfCompiler.enabled=true

spark.rapids.python.gpu.enabled=true
spark.rapids.python.memory.gpu.pooling.enabled=false
spark.rapids.sql.exec.ArrowEvalPythonExec=true
spark.rapids.sql.exec.MapInPandasExec=true
spark.rapids.sql.exec.FlatMapGroupsInPandasExec=true
spark.rapids.sql.exec.AggregateInPandasExec=true
spark.rapids.sql.exec.FlatMapCoGroupsInPandasExec=true
spark.rapids.sql.exec.WindowInPandasExec=true
```

Also modify 
```
spark.submit.pyFiles=/usr/lib/spark/jars/xgboost4j-spark_${SPARK_VERSION}-${XGBOOST_VERSION}-${SPARK_RAPIDS_VERSION}.jar,/usr/lib/spark/jars/rapids-4-spark_2.12-${SPARK_RAPIDS_VERSION}.jar
```

Starting 9/18, dataproc preview image ships spark 3.0.1 and AQE enabled, we probably need to test a bit before set 
```
spark.sql.adaptive.enabled=true
```

**Comment by sameerz** [comment]
@mengdong Pandas UDFs are a beta feature, I would not turn them on by default.

**Pull request closed without merging**